### ISSUE TITLE
[feature] Command to show quota. (possible using the /me endpoint)

### ISSUE THREAD

**Issue by eacp** [ipinfo/cli#82]
Sometime ago, I used the /me EP to validate the token during auth. The /me EP actually returns usefull quota information, but at this point it is not used.

I propose a command that displays the quota information.

```
$ ipinfo quota

Requests
Day: 0,
Month: 9,
Limit: 50000,
Remaining: 49991


"features": {
    "core": {
      "daily": 2147483647,
      "monthly": 50000
    },
    "hostio": {
      "daily": 2147483647,
      "monthly": 1000,
      "result_limit": 5
    }
  }
```

Or something similar. The design is pending.

**Comment by UmanShahzad** [comment]
Yeah a `quota` subcommand would be nice. But we probably need a different endpoint for this as the existing one gives potentially confusing data, like when it says `limit`/`remaining`, what does that mean - is that for `core`, `asn`, or what?

I'll figure smth out for this later, thanks for the suggestion!

**Comment by eacp** [comment]
@UmanShahzad  I have this proposal:

Either allow parameters in the quoata command to let the user choose WHAT to use, or default to core

**Comment by rm-Umar** [comment]
Solved in #131
